app-tests/_templates/next-template/src/pages/_app.js

```diff
@@ -1,7 +1,15 @@
+import React from 'react';
+import pt from 'prop-types';
+
 import '@espressive/legacy-css';
 
-const MyApp = ({ Component, pageProps }) => {
-  return <Component {...pageProps} />;
+const propTypes = {
+  Component: pt.element,
+  pageProps: pt.shape({}),
```

Comment:
Any time we have a real need for `object` as a rule, we should use it instead of just using `shape` to trick the linter. This would be a good scenario to use `object` and then comment `// eslint-disable-next-line react/forbidden-proptypes -- We do not know what the object params might be in this case`

Reply:
Yes, there were a lot of components with no props at all, I used shape for the sake of speed. We spend too much time trying to figure out what the prop types should be. We should add a rule for preventing us from adding components without prop types.

Reply:
Technically you just put that rule into effect with this eslint rule. ;)

Reply:
This empty shape prop was missed.